src/main/resources/translations/program_strings_en.properties

```diff
@@ -461,128 +461,127 @@ rph.donotuse = Do not use
 rph.cityname = City name
 rph.iso = ISO
-rph.foclength35mm = Focal length in 35mm format
+rph.foclength35mm = Equivalent 35 mm focal length
 rph.orgfilename = Original filename
-rph.duplicatestext = In case your chosen options lead to duplicate names, the tool will automatically autonumber the images.<br>Please specify how the tool should do this.
+rph.duplicatestext = In case your chosen options leads to duplicate names, the tool auto-numbers the images.<br>Please specify how the tool should do this.
 rph.noofdigits = No. of digits
-rph.nooffidgitsone = Start counting on 1st image
-rph.nooffidgitstwo = Start counting on 2nd image
+rph.nooffidgitsone = Start counting from 1st image
+rph.nooffidgitstwo = Start counting from 2nd image
 rph.filextension = File extension:
 rph.fileextleave = Leave as is
-rph.fileextlower = Make lower case
-rph.fileextupper = Make upper case
+rph.fileextlower = Make lowercase
+rph.fileextupper = Make uppercase
 rph.nameexample = Name: Prefix_suffix.extension
-rph.locateimgfolder = Locate the image folder ...
+rph.locateimgfolder = Locate the image folder…
 rph.noimgsnopathtitle = No path or selected files
 rph.noimgsnopathtext = No image folder path selected<br>and/or<br>No images selected in the main screen
 rph.selectedoptionstitle = Selected options
 rph.youselected = You selected:
 rph.asprefix = As prefix:
 rph.assuffix = And as suffix:
-rph.forcounting = For counting:
+rph.forcounting = For enumerating:
 rph.forextension = For the extension:
 
 #### dialog export metadata
-emd.toptext = This option gives you the possibility to export the metadata from your selected photo(s). A number of formats is supported.<br>\
-All formats give an export file per selected photo, apart from csv which will give you one (big) csv file for all selected photos.<br><br>\
-The exported files will be saved to the image folder you loaded your images from.
+emd.toptext = This option lets you export the metadata from your selected photo(s).<br>\
+All supported formats give an export file per selected photo, apart from CSV, which gives you one (big) CSV file for all selected photos.<br><br>\
+The exported files are saved to the image folder you loaded your images from.
 emd.exportall = Export all metadata
-emd.exportexif = Export exif data
-emd.extportxmp = Export xmp data
-emd.exportgps = Export gps data
-emd.exportiptc = Export iptc data
+emd.exportexif = Export Exif data
+emd.extportxmp = Export XMP data
+emd.exportgps = Export GPS data
+emd.exportiptc = Export IPTC data
 emd.exporticc = Export ICC data
-emd.uselang = Use language from Preferences for metadata tag descriptions
+emd.uselang = Use language from Preferences for metadata-tag descriptions
 emd.exportto = Export to:
-emd.txt = txt
+emd.txt = TXT
 emd.tab = tab
-emd.xml = xml
-emd.html = html
-emd.xmp = xmp
-emd.csv = csv
+emd.xml = XML
+emd.html = HTML
+emd.xmp = XMP
+emd.csv = CSV
 emd.allmetadata = All metadata<br><br>
 emd.dlgtitle = You want to export metadata
```

Comment:
Could this be explained?

Reply:
This would be much better when using "You selected the following metadata for export", or "You want to export the following metadata".
However, it is the title of an auto-sizing dialog box and the "better" titles would be too long.